### DISCUSSION TITLE
Welcome to vue3-carousel Discussions!

### DISCUSSION THREAD

**Discussion by ismail9k** [ismail9k/vue3-carousel#32] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by antssanjeewa** [comment]
Can the current slide bind to the v-model

**Reply by antssanjeewa** [reply]
I thank you for considering this.

Now can Bind model in update 0.1.20

**Comment by kostya110393** [comment]
Hey guy! I would like to ask you to make I can to turn off or turn on scrollTouch, scrollKey, scrollMouse and so on. Also I would like to hidden the left button if Carousel on first element and right button too if Carousel on last element

**Reply by ismail9k** [reply]
There's already a feature request open for this check #59
Also, check this to hide last button #45

**Reply by kostya110393** [reply]
> Для этой проверки уже открыт запрос функции [# 59](https://github.com/ismail9k/vue3-carousel/issues/59)
> Также отметьте это, чтобы скрыть последнюю кнопку [# 45](https://github.com/ismail9k/vue3-carousel/issues/45)

Thanks! I could hide button, but i can't turn off touch. I put on `:enableTouch="false"` but it doesn't work unfortunately
![image](https://user-images.githubusercontent.com/26055433/120477856-ab9b7e00-c3c5-11eb-9926-26604ed687eb.png)

**Reply by ismail9k** [reply]
Yes, As I mentioned it's an open feature request, which I will try to introduce soon ;)

**Reply by kostya110393** [reply]
> Yes, As I mentioned it's an open feature request, which I will try to introduce soon ;)

Would you like to add in addons "itemsToShow"? I saw that itemToShow don't update in props. Or can I get access "config"?
![image](https://user-images.githubusercontent.com/26055433/120749975-f54bac00-c51e-11eb-8c6c-05ce012edd07.png)

**Reply by ismail9k** [reply]
No updating props should update the current configs, I will open a bug issue for this

**Comment by suraj-shyplite** [comment]
How can I stop sliding on hover?

**Comment by oneWaveAdrian** [comment]
Hello, what's the timeline for the roadmap? I am keenly waiting on the very next checkbox -> being able to style the active/inactive slides to create a coverflow effect etc. I ❤️  the stability/minimalism on this component!

**Comment by Wixi105** [comment]
Hello, I am new to using this and I don't really know how to use it. Can someone direct me to a tutorial? I am trying to create a carousel of slides.

**Reply by oneWaveAdrian** [reply]
Have you seen the official documentation? https://ismail9k.github.io/vue3-carousel/
